### PR TITLE
[onert/lib] Update circle_traininfo schema

### PR DESCRIPTION
**Pull request by zetwhite**
This PR updates circle_traininfo schema to sync with inner source.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

### CODE REVIEW

File: runtime/libs/circle-schema/include/circle_traininfo.fbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/e94b2b8956bcc316bef19dc4885a5027f545845a/spec/circle_training.fbs
+// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/5dd9665b3dab4652a0afed50da86bf6964a41bdd/spec/circle_training.fbs
```
Comment (zetwhite) [inline runtime/libs/circle-schema/include/circle_traininfo.fbs:1]: ### background

This PR is to sync with inner source schema - which is lately updated: https://github.sec.samsung.net/one-project/circle-x/commit/5dd9665b3dab4652a0afed50da86bf6964a41bdd 

### Notes 
After this syncing, I will remove the schema file in inner source and remove this commit number here.
I have a plan to maintain `circle_traininfo.fbs` here, not in inner source.  
(related inner source issue [:link:](https://github.sec.samsung.net/one-project/circle-x/issues/57)

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-android-release onert-cross-aarch64-release onert-cross-debug tizen-gbs

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-android-release onert-cross-aarch64-release onert-cross-debug tizen-gbs

**Comment by hseok-oh** [comment]
Please rebase to resolve test fail on `onert-cross-debug`.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**